### PR TITLE
Update Rule “the-best-xamarin-resources/rule”

### PR DESCRIPTION
**Pull request by AntPolkanov**
Automatically generated by Netlify CMS

### CODE REVIEW

File: rules/the-best-xamarin-resources/rule.md [hunk 1]
```diff
@@ -1,55 +1,55 @@
...
-guid: b30559d3-4656-4cda-b1a2-fc1637b435c7
+title: Do you know where to find the best .NET MAUI resources?
 uri: the-best-xamarin-resources
-created: 2020-10-08T23:13:23.0000000Z
 authors:
-- title: Adam Cogan
-  url: https://ssw.com.au/people/adam-cogan
-- title: Matt Goldman
-  url: https://ssw.com.au/people/matt-goldman
+  - title: Adam Cogan
```
Comment (matt-goldman) [inline rules/the-best-xamarin-resources/rule.md:6]: Add yourself here

File: rules/the-best-xamarin-resources/rule.md [hunk 2]
```diff
@@ -1,55 +1,55 @@
...
 
 <!--endintro-->
 
 ### General
 
-**[Awesome Xamarin Forms](https://github.com/jsuarezruiz/awesome-xamarin-forms)** is a curated list on GitHub of a bunch of resources for Xamarin. This includes controls, effects, behaviors, tools, and more. It's a good 'one-stop-shop' for everything Xamarin. 
+**[Awesome .NET MAUI](https://github.com/jsuarezruiz/awesome-dotnet-maui)** (evolution of [Awesome Xamarin Forms](https://github.com/jsuarezruiz/awesome-xamarin-forms)) is a curated list on GitHub of a bunch of resources for .NET MAUI. This includes controls, effects, behaviors, tools, and more. It's a good 'one-stop-shop' for everything related to .NET MAUI. 
 
-**[Weekly Xamarin](http://weeklyxamarin.com/)** is a weekly newsletter sent every Friday or Saturday. It's a community-curated list of all the latest Xamarin news, blog posts, and podcasts.
+**[Weekly Xamarin](http://weeklyxamarin.com/)** is a weekly newsletter sent every Friday or Saturday. It's a community-curated list of all the latest .NET MAUI and Xamarin news, blog posts, and podcasts.
```
Comment (matt-goldman) [inline rules/the-best-xamarin-resources/rule.md:26]: This isn't around anymore, remove this and add Sands of MAUI

File: rules/the-best-xamarin-resources/rule.md [hunk 3]
```diff
@@ -1,55 +1,55 @@
...
 
 ### Learning
 
+The best video courses for learning .NET MAUI and Xamarin are **[Asfend Yar Hamid's real-world app series on Udemy](https://www.udemy.com/user/asfendyar)**.
```
Comment (matt-goldman) [inline rules/the-best-xamarin-resources/rule.md:30]: Not sure I can vouch for these anymore as I haven't seen them. They might be ok?

File: rules/the-best-xamarin-resources/rule.md [hunk 4]
```diff
@@ -1,55 +1,55 @@
...
 
-The best video courses for learning Xamarin are **[Asfend Yar Hamid's real-world app series on Udemy](https://www.udemy.com/user/asfendyar)**.
-
-The best book for learning Xamarin is **[Xamarin.Forms Projects** by Daniel Hindrikes and Johan Karlsson](https://www.packtpub.com/product/xamarin-forms-projects-second-edition/9781839210051). 
+The best book for learning .NET MAUI is **[.NET MAUI in Action by Matt Goldman](https://www.manning.com/books/dot-net-maui-in-action?utm_source=goforgoldman&utm_medium=affiliate&utm_campaign=book_goldman_dot_5_10_22&a_aid=goforgoldman&a_bid=38933097)** and [Xamarin.Forms Projects by Daniel Hindrikes and Johan Karlsson](https://www.packtpub.com/product/xamarin-forms-projects-second-edition/9781839210051) for learning Xamarin.
```
Comment (matt-goldman) [inline rules/the-best-xamarin-resources/rule.md:32]: 3rd edition comes out in Jan :) (.NET MAUI projects). Should we remove for now and update then? Or link to the book page and say it's one to look out for?

File: rules/the-best-xamarin-resources/rule.md [hunk 5]
```diff
@@ -1,55 +1,55 @@
...
+**[Telerik](https://www.telerik.com/maui-ui):** Telerik provides a range of controls focused on enterprise and business applications.
 
-**[Telerik](https://www.telerik.com/xamarin-ui):** Progress has made their Telerik UI available for Xamarin.Forms. Telerik provides a range of controls focused on enterprise and business applications. 
+**[DevExpress](https://www.devexpress.com/maui/):** DevExpress provides a library of high-performant, enterprise-grad controls for mobile .NET MAUI applications. There is a free option for registered users.
 
 ### Live Streams
 
 
-The best Xamarin stream is  [Kym Philpott's weekly stream on Twitch](https://www.twitch.tv/kymphillpotts).
+The best .NET MAUI and Xamarin stream is [Kym Philpott's weekly stream on Twitch](https://www.twitch.tv/kymphillpotts).
```
Comment (matt-goldman) [inline rules/the-best-xamarin-resources/rule.md:53]: He doesn't stream anymore :(

**Review by matt-goldman — COMMENTED** [review]
Good update! Needs some minor tweaks.

File: rules/the-best-xamarin-resources/rule.md (hunk 1 above)
Comment (AntPolkanov) [reply rules/the-best-xamarin-resources/rule.md:6]: Added

File: rules/the-best-xamarin-resources/rule.md (hunk 3 above)
Comment (AntPolkanov) [reply rules/the-best-xamarin-resources/rule.md]: Replaced with Gerald Versluis and James Montemagno.

File: rules/the-best-xamarin-resources/rule.md (hunk 4 above)
Comment (AntPolkanov) [reply rules/the-best-xamarin-resources/rule.md]: As per out conversation I added link to the 3rd edition which will come out in March 2024 and commented it out for now.

File: rules/the-best-xamarin-resources/rule.md (hunk 2 above)
Comment (AntPolkanov) [reply rules/the-best-xamarin-resources/rule.md]: Fixed

File: rules/the-best-xamarin-resources/rule.md (hunk 5 above)
Comment (AntPolkanov) [reply rules/the-best-xamarin-resources/rule.md]: Removed

**Review by matt-goldman — APPROVED** [review]

**Pull request merged**